app/src/pages/project/ProjectPage.tsx

```diff
@@ -37,16 +38,20 @@ const mainCSS = css`
 `;
 
 export function ProjectPage() {
+  const { projectId } = useParams();
   const data = useLazyLoadQuery<ProjectPageQuery>(
     graphql`
-      query ProjectPageQuery {
+      query ProjectPageQuery($projectId: GlobalID!) {
         ...SpansTable_spans
         ...TracesTable_spans
         ...ProjectPageHeader_stats
         ...StreamToggle_data
       }
     `,
-    {},
+    {
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      projectId: projectId!,
```

Comment:
I think if you do as string you can just cast it

Reply:
addressed in 20cf35aa